telcell/utils/transform.py

```diff
@@ -115,53 +116,80 @@ def filter_delay(paired_measurements: List[MeasurementPair],
             if x.time_difference <= max_delay]
 
 
-def sort_pairs_based_on_rarest_location(
+def categorize_measurement_by_coordinates(measurement: Measurement) -> Any:
+    return f'{measurement.lon}_{measurement.lat}'
+
+
+def categorize_measurement_by_rounded_coordinates(measurement: Measurement) -> Any:
+    warnings.warn("rounded coordinates imply odd shaped regions and should not be used for categorization")
```

Comment:
de logger wordt nog niet heel fanatiek gebruikt, maar deze zou daar wel in kunnen